magenta-lib/src/main/scala/magenta/deployment_type/CloudFormation.scala

```diff
@@ -171,7 +171,11 @@ class CloudFormation(vcsUrlLookup: VcsLookup) extends DeploymentType with CloudF
 
     // The tag name here ('gu:repo') and format ('guardian/:reponame') MUST
     // mirror the tag name and format used by @guardian/cdk.
-    val guRepoTag = ("gu:repo" -> vcsUrlLookup.get(target.parameters.build.projectName, target.parameters.build.id))
+    val vcsUrl = vcsUrlLookup.get(target.parameters.build.projectName, target.parameters.build.id)
+    val guRepoTag = ("gu:repo" -> vcsUrl)
+    if (vcsUrl == "") {
+      reporter.warning("Unable to detect the Github repository for your build. DevX require this information to provide better reporting for teams as part of the upcoming Service Catalogue. This info is probably missing because you are using an unsupported library (or custom script) to generate your Riffraff bundle. To fix this, we recommend that you use https://github.com/guardian/actions-riff-raff/ or, if you are using https://github.com/guardian/sbt-riffraff-artifact or https://github.com/guardian/node-riffraff-artifact, upgrade to the latest version.")
```

Comment:
Interested to see how this renders as it's quite a long message. If we place a break before the remediation steps, might it make it more obvious?

Reply:
Alternative could be to link to an external resource, similar to the [default branch warning](https://github.com/guardian/riff-raff/blob/06cee9eaa23648d197689ae3b0351fc417e326f6/riff-raff/app/deployment/actors/DeployGroupRunner.scala#L179).